designate/central/service.py

```diff
@@ -806,7 +806,8 @@ def create_zone(self, context, zone):
                 zone.parent_zone_id = parent_zone.id
                 # Do subzone policy check instead of regular create_zone
                 policy.check('create_sub_zone', context, target)
-            else:
+            elif not policy.check('create_super_zone', context,
+                                do_raise=False):
```

Comment:
nit: please change the spacing for the do_raise=False to match the colon ( ).
https://peps.python.org/pep-0008/